kuka_robot_descriptions/package.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>kuka_robot_descriptions</name>
+  <version>0.0.1</version>
+  <description>Robot desciptions for KUKA robots</description>
+  <author>Áron Svastits</author>
+  <maintainer email="svastits1@gmail.com">Áron Svastits</maintainer>
+  <license>BSD</license>
```

Comment:
Suggestion: switch to using [SPDX license identifiers](https://spdx.org/licenses).

They are machine readable, easier to verify (by automated tools) and leave no room for ambiguity (*which* variant of BSD is this fi?).

Reply:
Initially this repo was ported from ros-industrial, where the support packages had BSD licenses, and we did not want to modify them. 
Thanks for your suggestion, we will consider using SPDX - there should be a licensing decision for the whole kroshu organization in the (close) future, after which we will update all licenses 
So my preferred way would be to create an issue, write down your suggestion and we will come back to licensing in a later PR

Reply:
> Initially this repo was ported from ros-industrial, where the support packages had BSD licenses, and we did not want to modify them.

in that case all `BSD` license references would be `BSD-3-Clause`.

> Thanks for your suggestion, we will consider using SPDX - there should be a licensing decision for the whole kroshu organization in the (close) future, after which we will update all licenses So my preferred way would be to create an issue, write down your suggestion and we will come back to licensing in a later PR

Sure. I just wanted to refer you to them.

re: licensing in general: `Apache-2.0` and `MIT` would be fully compatible with `BSD-3-Clause`.